WakaTime.py

```diff
@@ -8,7 +8,7 @@
 ==========================================================="""
 
 
-__version__ = '11.0.8'
```

Comment:
let's bump the version in a separate commit

Reply:
here's the [commit](https://github.com/wakatime/sublime-wakatime/pull/117/commits/67ca7d929a463337bb9e22bd308ef19420998a68)